tools/agents/fleet_budget.py

```
@@ -22,43 +22,107 @@
+    try:
+        return json.load(io.open(PROVIDERS, encoding="utf-8"))
+    except (OSError, ValueError):
+        return {}
+
+
+def deployed_models():
+    """מחרוזות המודל שבאמת מוגדרות ב-litellm.yaml.
+
+    הגרסה הראשונה של `caps()` הייתה **רשימה ידנית** של ארבעה מודלים.
+    ספק שנוסף לקונפיג — ‏NIM, ‏Mistral, ‏OpenRouter — פשוט לא הופיע
+    בה, ולכן לא נספר **ולא דווח שאינו נספר**: הוא נראה בדיוק כמו
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)